colabs/scikit/decision-trees.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    run = wandb.init(project='...', entity='...', config = config)
</pre><p>Should we remove the <code>entity</code>  argument and provide some dummy project name here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I plan to do this with all the "Fundamental Guides". I'll create another team and put that `entity` afterwards.

Reply:
Then you will have to open the project so that anyone can submit a run there.

Reply:
Good point, would you suggest anony mode instead ? or should I just add my id ?

Reply:
I don't think we should use anonymous mode here. I think anyone who will use our examples colab will have the desire to create an account of his/her own. How about we just provide a dummy project name without entity. That way when someone runs the colab that person will create a new project in his own entity. 

With anonymous mode there's an extra step of claiming the run if someone is interested. For a first timer it might be confusing. etc.